source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs

```diff
@@ -1245,5 +1245,155 @@ public void TestParsingRichTextBlockWithInvalidInlineType()
 
             StringAssert.Contains(ex.Message, "Property 'type' must be 'TextRun'");
         }
```

Comment:
nit: newline here to break between functions

Reply:
thanks

---
In reply to: [341284053](https://github.com/microsoft/AdaptiveCards/pull/3548#discussion_r341284053) [](ancestors = 341284053)